bids/config.py

```diff
@@ -71,7 +70,7 @@ def from_file(filenames, error_on_missing=True):
     if isinstance(filenames, str):
         filenames = [filenames]
     for f in filenames:
-        if exists(f):
+        if Path(f).exists():
```

Comment:
TBH, I don't see the advantage here. `os.path.exists()` will already work with `os.PathLike`s.

Reply:
I think you are right, no advantage other than the consistent use of `pathlib` in favor of `os.path` where possible. Nut sure if it is desirable.

Should I revert this change?

Reply:
I'm fine w/ going all in on pathlib on this one. Just for consistency's sake. No need to import from `os`.